### PR TITLE
add License field + Copyright section to all CPSs

### PR DESCRIPTION
**Pull request by rphair**
Implements change agreed in #762 for all merged CPSs.  These all add CC licensing (vs. Apache, so far) since, as mainly will be true for CPSs, they're all narrative & no code.

This is also a check to see if there will be any downstream problems, mainly due to the new mandatory field in the YAML preamble, on (cc @fill-the-fill @katomm):
- `cips.cardano.org` ([cf-cip-frontend](https://github.com/cardano-foundation/cf-cip-frontend))
- the CIP / CPS build script on the Dev Portal

TODO wait before merging to confirm no downstream problems before:
- [ ] merging this PR
- [ ] updating PRs for other CPSs in progress

### CODE REVIEW

**Comment by Ryun1** [comment]
Pleased to see this
will wait on confirmation from downstream tooling developers that they are aware, before I approve

**Comment by rphair** [comment]
Note I've just added the license stuff to CIPs 11 & 13 (merged earlier today) in commit 8c244a8 but they don't appear in the PR thread or diffs yet (i.e. the sometime "git PRs out of sync" issue which sometimes last a few hours, very rarely).  Rest assured they are there & they can be seen in the PR branch. 😇

**Comment by rphair** [comment]
Question to pre-empt any difficulties has been more officially presented to the site building team in https://github.com/cardano-foundation/developer-portal/issues/1221 ... if not hearing for another week then I'd say we can go ahead and merge this since my educated guess is that the extra field & section will be ignored.  And if they _were_ to cause site building errors then we would have an additional imperative to get the build scripts updated quickly. 😇

**Review by Ryun1 — APPROVED** [review]
Nice improvement ✅

**Comment by rphair** [comment]
thanks & without any objection from the CF Dev Portal representatives (@fill-the-fill @katomm), as well as a preliminary assessment that we didn't have a Dev Portal build failure the last time we had conditions like this (https://github.com/cardano-foundation/developer-portal/issues/1221#issuecomment-1973574971), I think we have done due diligence before merging this change.

Any opinions otherwise, please blow the whistle now... otherwise I'll merge this as soon as we get a another confirmatory review from @Crypto2099 (cc @KtorZ @SebastienGllmt).

File: CPS-0005/README.md
Comment (Crypto2099) [inline CPS-0005/README.md:174]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

???

File: CPS-0007/README.md
Comment (Crypto2099) [inline CPS-0007/README.md:104]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

File: CPS-0010/README.md
Comment (Crypto2099) [inline CPS-0010/README.md:361]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

File: CPS-0011/README.md
Comment (Crypto2099) [inline CPS-0011/README.md:69]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

File: CPS-0013/README.md
Comment (Crypto2099) [inline CPS-0013/README.md:106]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

**Review by Crypto2099 — APPROVED** [review]
Probably a quick oversight from copying from the CIP template, but update CIP == CPS in the Copyright block but otherwise everything looks good to me!

**Comment by rphair** [comment]
thanks @Crypto2099 for catching the above... I'm grateful not to have left people wondering about that & then have had to go back & fix all of them later 🥴

**Pull request merged**
